engine/flink/avro-components-utils/src/test/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/serialization/ConfluentKafkaAvroSerializationSpec.scala

```diff
@@ -103,10 +105,18 @@ class ConfluentKafkaAvroSerializationSpec extends KafkaAvroSpecMixin with TableD
   private def runSerializationTest(table: TableFor5[SchemaRegistryProviderSetup, Option[Schema], GenericRecord, Any, String], version: Option[Int], schemas: List[Schema]): Assertion =
     forAll(table) { (providerSetup: SchemaRegistryProviderSetup, schemaForWrite: Option[Schema], givenObj: GenericRecord, expectedObj: Any, topic: String) =>
       val topicConfig = createAndRegisterTopicConfig(topic, schemas)
-      val serializer = providerSetup.provider.serializationSchemaFactory.create(topicConfig.output,
-        schemaForWrite.map(s => new NkSerializableParsedSchema[ParsedSchema](new AvroSchema(s))), kafkaConfig)
+      val serializer = providerSetup.provider.serializationSchemaFactory.create(topicConfig.output, schemaForWrite.map(s => toRuntimeSchemaData(topicConfig.output, s)), kafkaConfig)
 
       providerSetup.pushMessage(serializer, givenObj, topicConfig.output)
+
+      if(schemaForWrite.isDefined)
+        kafkaClient.createConsumer()
+          .consumeWithConsumerRecord(topicConfig.output).take(1)
+          .foreach(_.headers().toArray.map(_.key()) should contain oneOf(KeySchemaIdHeaderName, ValueSchemaIdHeaderName))
```

Comment:
Why oneOf? Aren't we sure what will be there?

Reply:
right, it's always `Value..` for now